modules/smart-agent_health-checker/conf/00-health-checker-value.yaml

```diff
@@ -0,0 +1,14 @@
+module: health-checker
+name: value
+id: health_checker_value
+
+transformation: true
+aggregation: false
```

Comment:
It is better to provide a variable to let the user customize this depending on his environment.

Reply:
done